datalad/interface/tests/test_run_procedure.py

```diff
@@ -258,7 +258,7 @@ def test_configs(path):
     # config on dataset level:
     ds.config.add(
         'datalad.procedures.datalad_test_proc.call-format',
-        'python "{script}" "{ds}" local {args}',
+        'python {script} {ds} local {args}',
```

Comment:
Here and above we should also use `sys.executable` not `python`.

Reply:
oh right. Thanks!